test/jest/unit/iac-unit-tests/file-parser.spec.ts

```diff
@@ -64,4 +65,13 @@ describe('parseFiles', () => {
 
     await expect(parseFilesFn).rejects.toThrow(UnsupportedFileTypeError);
   });
+
+  it('will not parse if Helm file', async () => {
```

Comment:
nit (although suggested to move this logic inwards):
```suggestion
  it('does not call the kubernetes parser if detected a Helm file', async () => {
```

+can we add an assertion/another-test for the specific error throwing in case of a helm file?

Reply:
changed this to reflect the implementation